app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java

```diff
@@ -0,0 +1,84 @@
+package com.example.harvestfresh.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.harvestfresh.Cart;
+import com.example.harvestfresh.CartAdapter;
+import com.example.harvestfresh.R;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartFragment extends Fragment {
+
+    private static final String TAG = "CartFragment";
+    private static final String PRODUCT_DISPLAY = "Cart ";
+
+    private RecyclerView rvCart;
+    private CartAdapter fragmentAdapter;
+    private List<Cart> allCarts;
+
+    public CartFragment() {
+    }
+
+    public static CartFragment newInstance() {
```

Comment:
Why is this necessary when the constructor is public?

Reply:
This was something done by default by android studio when generating a new fragment, fixed!